extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxRenderServiceHelper.php

```diff
@@ -120,11 +144,29 @@ public function isValidHeroDataItem( $item, $heroData ) {
 	 * required for testing mobile template rendering
 	 * @return bool
 	 */
-	public function isWikiaMobile() {
+	public function isMobile() {
 		return \F::app()->checkSkin( 'wikiamobile' );
 	}
 
 	/**
+	 * @return bool
+	 */
+	public function isMercury() {
```

Comment:
mercury is the only one using articleasjson currently 
but that might not be always the case
I'd rename it to isArticleAsJson or something like that